client/site-profiler/components/styles-v2.scss

```diff
@@ -323,6 +323,41 @@
 					color: #fff;
 				}
 			}
+
+			.metrics-insight-detailed-content {
+				margin-top: 42px;
+
+				table {
+					table-layout: fixed;
```

Comment:
A fixed layout for the table will keep all columns of similar size, which we don't want since it looks way off. I think using work breaks on `td` would resolve that.